ai-player.js

```diff
@@ -345,6 +345,13 @@ function createAiPlayer(game, options) {
         return opponents.length == 1;
     }
 
+    function randomizeChoice() {
```

Comment:
This function could do with a comment explaining what it's for and why it behaves differently in the end game.